packages/components/pie-modal/src/modal.scss

```diff
@@ -56,7 +58,15 @@
 
     & .c-modal-closeBtn {
         position: absolute;
-        right: var(--dt-spacing-d);
-        top: var(--dt-spacing-d);
+        inset-inline-end: var(--dt-spacing-d);
+        inset-block-start: var(--dt-spacing-d);
     }
 }
+
+::backdrop {
+  --color-backdrop: rgb(0,0,0,0.5);
```

Comment:
I've tried to use the token `--dt-color-overlay:` but doesn't work for some reason, I bumped the token package and tried but the colour doesn't come through. Thoughts?

Reply:
Just checking you tried it with `var(--dt-color-overlay)`?

Reply:
Yup tried that.

Reply:
I've just pulled down your PR and had a play around. One thing you can do (which saves the extra definition) is:

```
// Add this to the top of the modal.scss
@use '@justeat/pie-design-tokens/dist/jet.scss' as dt;

// and then for the definition…
--color-backdrop: #{dt.$color-overlay};
```

So that'll use sass to add the variable into the backdrop context in the web component. I tried all sorts of other ways of doing it just with native CSS, but I couldn't find a way of doing it. For some reason, `::backdrop` won't recognise the CSS variable unless it's set to a string inside the web component style context. I'm guessing it's to protect against a niche use case with shadow dom 🤷

Reply:
Cheers both appreciate you looking at it 👍